src/test/tasks/tasks.yaml

```diff
@@ -96,3 +96,6 @@ tasks:
     actions:
       - task: foo:foobar
       - task: remote:echo-var
+  - name: extra-foobar
```

Comment:
Does any test use this new `extra-foobar` task?

Reply:
lol my local debugger, let me create that test

Reply:
pushed a test for the rest of the world lol